app/assets/stylesheets/static.scss

```diff
@@ -1,3 +1,30 @@
 // Place all the styles related to the static controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
+.branding {
+  p {
+    min-height: 24px;
+    margin: 24px;
+
+    .left {
+      float: left;
+      font-weight: bold;
+    }
```

Comment:
In the future we should replace this with grid or flex. It'll be less effort to handle different window sizes.

Reply:
That's a good comment for the github issue on CSS implements!